### PR TITLE
Creating a PR for DC-Portal Initial Template UI

### PR DESCRIPTION
**Pull request by paskantishubham**
This PR address to #6 

* index.html- Loading react, bootstrap,jquery,etc through cdn.
* src/App.js- Loads UI
* src/Components/Sidebar.js- Created 3 parts of sidebar, 
                    1-Header for GE logo, Title, User Info
                    2-Body for Menu Options
                    3-Footer for copywrite
* src/Components/CenterHeader.js- Used for Search Bar.
* Rest files are just templates which will be used in future.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+    <!-- Modular library -->
+    <script
+      type="text/javascript"
+      src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/0.19.31/system.js"
+    ></script>
+
+    <title>DC-portal</title>
+
+    <style>
```
Comment (dkrypt) [inline index.html:32]: Please maintain a separate CSS file which is referenced from `index.html`

**Review by dkrypt — CHANGES_REQUESTED** [review]
1. Please setup a linter for JS. This makes the styling stricter and uniform. A popular choice is `eslint.js`.
2. Please provide a descriptive README for how to get started. For a beginner I want to be able to run the code without hassle. A good documentation is the key to any new project.

**Comment by paskantishubham** [comment]
> 1. Please setup a linter for JS. This makes the styling stricter and uniform. A popular choice is `eslint.js`.
> 2. Please provide a descriptive README for how to get started. For a beginner I want to be able to run the code without hassle. A good documentation is the key to any new project.

File: index.html (hunk 1 above)
Comment (paskantishubham) [reply index.html:39]: I have made these changes, will be releasing in next PR.

**Comment by ayasuda-ge** [comment]
@paskantishubham please rename the “assets” to “webui-assets” for backend integration purpose.

**Review by ayasuda-ge — CHANGES_REQUESTED** [review]
please rename the folder “assets” to “webui-assets” for integration purpose @paskantishubham

**Review by ayasuda-ge — CHANGES_REQUESTED** [review]
please rename “assets” to “webui-assets” for integration purposes. Please also ensure to leave all artifacts in the webui-assets. @paskantishubham

**Comment by paskantishubham** [comment]
I have renamed assets folder and did necessary changes to the code as requested by @ayasuda-ge 

@ayasuda-ge Please merge it

**Review by ayasuda-ge — CHANGES_REQUESTED** [review]
I also notice these folders- “font”, “styling” and index.html are all exposed outside of the “webui-assets” Please help refactor those to be under the “webui-assets” folder. Thanks @paskantishubham

**Comment by paskantishubham** [comment]
> I also notice these folders- “font”, “styling” and index.html are all exposed outside of the “webui-assets” Please help refactor those to be under the “webui-assets” folder. Thanks @paskantishubham

@ayasuda-ge I have restructured the webui-assets folder, please review it.

**Review by ayasuda-ge — APPROVED** [review]
perfect! Thank you @paskantishubham

**Pull request merged**